src/amitaRJun20/CheckCharacterMethods.java

```diff
@@ -0,0 +1,39 @@
+package amitaRJun20;
+
+public class CheckCharacterMethods {
+	void testCharMethod(String name,char ch) {
+		int digitCount=0;
+		int letterCount=0;
+		int upperCaseCount=0;
+		int lowerCaseCount=0;
+		int specialCharCount=0;
+		
+		for(int index=0;index<name.length();index++) {
+			if(Character.isDigit(name.charAt(index))) {
+				digitCount++;
+			}
+			if(Character.isLetter(name.charAt(index))) {
+				letterCount++;
+			}
+			if(Character.isUpperCase(name.charAt(index))) {
+				upperCaseCount++;
+			}
+			if(Character.isLowerCase(name.charAt(index))) {
+				lowerCaseCount++;
```

Comment:
Here so many comparisons happening, which is not required. 
Like if char is digit so it will go in your all if block. which is wrong right !!
So here you need to write if, else if....else block. 
Second point letterCount without using this variable you can print the expected output. Think about it.

Reply:
Thank you for your feedback....
after i committed the code then i realized after reading your comment few times that letter count variable not required....so again updated the logic and committed...sorry for multiple commits...

Reply:
No need to say sorry, it's your PR...you can make as many commits you want.